JS/Planes/MyPlane.js

```diff
@@ -1,11 +1,9 @@
-const Plane = require('./Plane');
+const Plane = require("./Plane");
```

Comment:
What is the purpose of storing `MyPlane` class?
It is not used anywhere in the code.